.travis.yml

```diff
@@ -2,8 +2,12 @@ language: rust
 
 rust:
     - stable
+    - nightly-2018-07-08
```

Comment:
Are we happy to take a dep on nightly rust here overall?

Reply:
Right, this is more a "heads up" that there may be an upcoming issue in nightly. I should probably add a flag to allow for failures and drop the date.